scripts/base/init-bare.zeek

```diff
@@ -4304,6 +4306,11 @@ export {
 		persistent:     bool;
 	};
 
+        type RDP::ClientClusterData: record {
+                flags:                  int;
+                redir_session_id:       int;
```

Comment:
Both these should be `count`.

Also adding documentation comments would be nice.